MagentoEcsFargate/magento_platform/magento_app_stack.py

```diff
@@ -0,0 +1,104 @@
+from re import A
+from aws_cdk import (
+    NestedStack,
+    aws_ec2 as ec2,
+    aws_ecs as ecs,
+    aws_elasticloadbalancingv2 as elbv2,
+    aws_ecs_patterns as ecs_patterns,
+    aws_secretsmanager as sm,
+    aws_certificatemanager as cm,
+    CfnOutput,
+    Duration,
+)
+
+from constructs import Construct
+
+
+class MagentoAppStack(NestedStack):
+
+    def __init__(self, scope: Construct, vpc, appSG, database_instance, es_domain, **kwargs):
+        super().__init__(scope, "MagentoAppStack", **kwargs)
+
+        # Create ECS cluster
+        _cluster = ecs.Cluster(self, "MagentoCluster", vpc=vpc)
+
+        # Retrieve DB secret ARN
+        dbsecret = sm.Secret.from_secret_complete_arn(self, "DatabaseSecret",
+                                                      secret_complete_arn=database_instance.secret.secret_arn,
+                                                      )
+        # Create application user password
+        secretMagentoUser =  sm.Secret(self,  'passwordMagentoUser', secret_name = "passwordMagentoUser");
+        
+        # Create an application load balancer.
+        # WARNING : This example is provided as a sample and rely on HTTP, which should never be used in production system. 
+        magento_lb = elbv2.ApplicationLoadBalancer(self, "MagentoLB", vpc=vpc, internet_facing=True, load_balancer_name="magento-lb") 
+       
+       # Create Fargate service running on an ECS cluster fronted by the application load balancer.
+        self.magento_service = ecs_patterns.ApplicationLoadBalancedFargateService(self, "MagentoService",
```

Comment:
Indentation of the parameters here look broken.  (at least line `image=`, and `subnet_type=`)
![image](https://user-images.githubusercontent.com/61999/169985015-945f0920-cfa8-49aa-9bf0-94bf7c43293e.png)